# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/libx11/hotfix-pragma.patch.avr32
# Copyright (C) 2021 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- libx11/src/xkb/XKBBind.c.vanilla	2021-10-07 00:42:26.913105211 +0200
+++ libx11/src/xkb/XKBBind.c	2021-10-07 00:43:08.943103535 +0200
@@ -227,14 +227,14 @@
 #pragma clang diagnostic push
 #pragma clang diagnostic ignored "-Wdeprecated-declarations"
 #elif defined(__GNUC__)
-#pragma GCC diagnostic push
-#pragma GCC diagnostic ignored "-Wdeprecated-declarations"
+//#pragma GCC diagnostic push
+//#pragma GCC diagnostic ignored "-Wdeprecated-declarations"
 #endif
     return XKeycodeToKeysym(dpy, event->keycode, col);
 #ifdef __clang__
 #pragma clang diagnostic pop
 #elif defined(__GNUC__)
-#pragma GCC diagnostic pop
+//#pragma GCC diagnostic pop
 #endif
 
 }
